modules/nops/sparc/random.rb

```diff
@@ -69,44 +69,42 @@ class MetasploitModule < Msf::Nop
 
   def initialize
     super(
-      'Name'        => 'SPARC NOP Generator',
-      'Alias'       => 'sparc_simple',
+      'Name' => 'SPARC NOP Generator',
+      'Alias' => 'sparc_simple',
       'Description' => 'SPARC NOP generator',
-      'Author'      => 'vlad902',
-      'License'     => MSF_LICENSE,
-      'Arch'        => ARCH_SPARC)
+      'Author' => 'vlad902',
+      'License' => MSF_LICENSE,
+      'Arch' => ARCH_SPARC)
 
     register_advanced_options(
       [
-        OptBool.new('RandomNops', [ false, "Generate a random NOP sled", true ])
-      ])
+        OptBool.new('RandomNops', [ false, 'Generate a random NOP sled', true ])
+      ]
+    )
   end
 
-
-
   # Nops are always random...
   def generate_sled(length, opts)
-
     badchars = opts['BadChars'] || ''
-    random   = opts['Random']   || datastore['RandomNops']
-    blen     = length
+    opts['Random'] || datastore['RandomNops']
```

Comment:
```suggestion
    _random = opts['Random'] || datastore['RandomNops']
```

Reply:
I've removed this line.

This module provides a `RandomNops` advanced module option, but ignores it (as explained by the existing code comment: "Nops are always random"). I've left it as is to prevent breaking any workflows which rely on this option.